plugin.go

```diff
@@ -167,6 +174,11 @@ func (c *Plugin) Run(cliConnection plugin.CliConnection, args []string) {
 		os.Exit(0) // Ignore CLI-MESSAGE-UNINSTALL etc.
 	}
 }
+
+func getGitRepoConfiguration(ac *cli.ArgConsumer) string {
```

Comment:
The YAGNI antennae is telling me that _currently_ the story is asking for the second argument to just be the URI of the Git repo to be added. Just sayin' 😉